### PR TITLE
New SDK update for Automation

### PR DESCRIPTION
**Pull request by vrdmr**
## Description
Contains changes introduced in the following spec changes:
https://github.com/Azure/azure-rest-api-specs/pull/3363
https://github.com/Azure/azure-rest-api-specs/pull/3420

Other changes:
- Enabled the Runbook tests and rerecorded them.
- Bumped up the version of the SDK - `3.2.0-preview`.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@vrdmr please update the PR with latest upstream changes

File: src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Automation Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.")]
 
-[assembly: AssemblyVersion("3.0.1.0")]
-[assembly: AssemblyFileVersion("3.1.0.0")]
+[assembly: AssemblyVersion("3.2.0.0")]
```
Comment (dsgouda) [inline src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs]: Please keep `AssemblyVersion` unchanged, only `AssemblyFileVersion` needs to be updated

File: src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Automation</PackageId>
     <Description>Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.</Description>
-    <Version>3.1.0-preview</Version>
+    <Version>3.2.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj]: Please update the `PackageReleaseNotes` 
Also, please make it descriptive similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj#L13)

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, please address the comments.

File: src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj (hunk 2 above)
Comment (vrdmr) [reply src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj]: Done.

**Comment by dsgouda** [comment]
@vrdmr Please merge commits into a single commit and we should be good to merge

**Comment by shahabhijeet** [comment]
@vrdmr this PR cannot be merged with the current commit history. Please either open a clean PR with require commits or squash/rebase the existing commit history. I would prefer a new PR with clean commit history.

**Comment by vrdmr** [comment]
@dsgouda @shahabhijeet Closing this PR in favor of #4583.

**Pull request closed without merging**